components/CostsTable.vue

```diff
@@ -13,7 +13,7 @@
       >
         <tr>
           <td class="ps-2">
-            {{ appStore.getCostLabel(childCost.id) }}
+            {{ appStore.getCostLabel(childCost.id) }} <span v-if="childCost.id === 'life_years'">*</span>
```

Comment:
Think this shouldn't have a space:

```suggestion
            {{ appStore.getCostLabel(childCost.id) }}<span v-if="childCost.id === 'life_years'">*</span>
```